bib/cmd/bootc-image-builder/main.go

```diff
@@ -156,14 +166,21 @@ func saveManifest(ms manifest.OSBuildManifest, fpath string) error {
 }
 
 func manifestFromCobra(cmd *cobra.Command, args []string) ([]byte, error) {
-	hostArch := arch.Current()
-	repos := loadRepos(hostArch.String())
+	buildArch := arch.Current()
+	repos := loadRepos(buildArch.String())
 
 	imgref := args[0]
 	rpmCacheRoot, _ := cmd.Flags().GetString("rpmmd")
 	configFile, _ := cmd.Flags().GetString("config")
 	tlsVerify, _ := cmd.Flags().GetBool("tls-verify")
 	imgType, _ := cmd.Flags().GetString("type")
+	targetArch, _ := cmd.Flags().GetString("target-arch")
```

Comment:
We should be handling "target-variant" as well.

Reply:
Happy to add more here - but pardon my ignorance, what does `target-variant` mean in this context?

Reply:
Some Arm platforms have multiple variants.  So if BIB is calling skopeo or podman then it is needs to pass the variant as well as the Arch if the user specified a variant.

Reply:
I added a TODO here for now, if you could share some container or any other link where I can inspect this a bit more that would be super helpful. I will then add/fix this in a followup.